snap/snapcraft.yaml

```diff
@@ -73,11 +71,12 @@ parts:
 
       version="$(python3 setup.py --version)"
       ${SNAP}/libexec/snapcraft/craftctl set version="$version"
-  
+      [ -n "$(echo $version | grep "+git")" ] && grade=devel || grade=stable
```

Comment:
someone needs to double-check this but I don't think the version has `+git`, but `+g` (from the setuptools versioning scheme used in starbase)

Reply:
eg one of the dev files I have here is `rockcraft_9.9.9.post1+g5f30fc3_amd64.snap`

Reply:
The current main packed as `rockcraft_0.0.post330+g2ae4796_amd64.snap`.
But I also had some old snaps like `rockcraft_0+git.2c79cb7_amd64.snap` and `rockcraft_0+git.f51824d-dirty_amd64.snap`

Reply:
yeah,  I changed it to "post" and tried out locally, GH was down last night, so could not push anymore :sweat_smile: